chemprop/cli/train.py

```diff
@@ -450,9 +450,16 @@ def process_train_args(args: Namespace) -> Namespace:
         raise ArgumentError(
             argument=None, message=f"Input data must be a CSV file. Got {args.data_path}"
         )
+
     if args.output_dir is None:
         args.output_dir = Path(f"chemprop_training/{args.data_path.stem}/{NOW}")
 
+    if args.epochs != -1 and args.epochs <= args.warmup_epochs:
+        raise ArgumentError(
+            argument=None,
+            message=f"The number of epochs should be higher than the number of epochs during warmup. Got {args.epochs} epochs and {args.warmup_epochs} warmup epochs",
+        )
```

Comment:
this should be handled at the level of the `Lightning.Trainer`'s via the `fast_dev_run` argument

Reply:
So should we add another CLI argument to expose that to the user? I think it is simpler to let CLI users run for a single epoch. Though users can circumvent this argument error by `--warmup-epochs 1 --epochs 2`, so I guess I'm okay with leaving it all as is.

Reply:
yeah IMO it would be more idiomatic to add a `-n/--dry-run` flag that runs only a few (2--4) batches under the hood _without_ writing any files (logs, checkpoints, tensorboard, output files, etc.)

Reply:
I think if all of our outputs are stored under `output_dir`, then when the `--dry-run` flag is set, we can just override that to be `/dev/null`, silently writing everything to the ether.

Reply:
I like the idea of have a `-n/--dry-run` flag for a quick test. @KnathanM we can discuss this in the dev meeting and decide whether to open an PR for this. As we already provided an error message here, users can circumvent the issue you mentioned for now.